core/src/main/java/org/elasticsearch/node/InternalSettingsPreparer.java

```diff
@@ -123,8 +124,13 @@ static void initializeSettings(final Settings.Builder output, final Settings inp
 
     /**
      * Checks all settings values to make sure they do not have the old prompt settings. These were deprecated in 6.0.0.
+     * This check should be removed in 8.0.0.
      */
     private static void checkSettingsForTerminalDeprecation(final Settings.Builder output) throws SettingsException {
+        // This method to be removed in 8.0.0, as it was deprecated in 6.0 and removed in 7.0
+        if (Version.CURRENT.toString().startsWith("8")) {
```

Comment:
How about `Version.CURRENT.major == 8`? And I think maybe we should only make this an assertion?

Reply:
Thx, i wasnt aware exactly how to do it